.github/actions/hps_services/action.yml

```diff
@@ -43,15 +43,25 @@ runs:
         version: tags/${{ inputs.version }}
         token: ${{ inputs.token }}
 
-    - if: ${{ inputs.version == 'latest-dev' }}
-      uses: dsaltares/fetch-gh-release-asset@master
-      name: Download Internal Services Artifact
+#    - if: ${{ inputs.version == 'latest-dev' }}
+#      uses: dsaltares/fetch-gh-release-asset@master
+#      name: Download Internal Services Artifact
+#      with:
+#        file: 'docker-compose-internal.tar.gz'
+#        target: 'docker-compose-artifact/docker-compose-internal.tar.gz'
+#        repo: ansys-internal/rep-deployments@mpawlik/dt_integration
+#        version: tags/latest-dev
+#        token: ${{ inputs.token }}
+    - uses: dawidd6/action-download-artifact@v3
```

Comment:
I think we could in general use this pattern:
- for released versions we get the artifacts from the release assets
- for latest-dev, we get artifacts from the latest successful build

To support testing against features, we should expose a new input `feature` (only used when `version=latest-dev`) to be passed as  branch to dawidd6/action-download-artifact.